docs/cql/functions.rst

```diff
@@ -33,13 +32,16 @@ CQL supports two main categories of functions:
 - The :ref:`aggregate functions <aggregate-functions>`, which are used to aggregate multiple rows of results from a
   ``SELECT`` statement.
 
-.. In both cases, CQL provides a number of native "hard-coded" functions as well as the ability to create new user-defined
-.. functions.
+In both cases, CQL provides a number of native "hard-coded" functions as well as the ability to create new user-defined
+functions.
+
+.. note:: Although user-defined functions are sandboxed, protecting the system from a "rogue" function, user-defined functions are disabled by default for extra security.
+. See the ``enable_user_defined_functions``
+   in ``scylla.yaml`` to enable them.
 
-.. .. note:: By default, the use of user-defined functions is disabled by default for security concerns (even when
-..    enabled, the execution of user-defined functions is sandboxed and a "rogue" function should not be allowed to do
-..    evil, but no sandbox is perfect so using user-defined functions is opt-in). See the ``enable_user_defined_functions``
-..    in ``scylla.yaml`` to enable them.
+   Additionally, user-defined functions are still experimental, and need to be explicitly enabled by adding ``udfs`` to the list of
```

Comment:
I think the experimental feature name is `udf`, not `udfs`. Unless `udfs` also works, somehow? Can you please check?

Reply:
I'll change do `udf`.